src/Mono.Android/Android.Runtime/AndroidRuntime.cs

```diff
@@ -530,7 +531,7 @@ public void RegisterNativeMembers (JniType nativeClass, Type type, ReadOnlySpan<
 								}
 
 							if (minfo == null)
-								throw new InvalidOperationException (String.Format ("Specified managed method '{0}' was not found. Signature: {1}", mname.ToString (), signature.ToString ()));
+								throw new InvalidOperationException (FormattableString.Invariant ($"Specified managed method '{mname.ToString ()}' was not found. Signature: {signature.ToString ()}"));
```

Comment:
I assume this should *either* use `{mname}…{signature}` *or* use `{mname.ToString(CultureInfo.InvariantCulture)}…{signature.ToString(CultureInfo.InvariantCulture)`.  What is here will call `.ToString()` with whatever is the current culture, which is most likely *not* the Invariant culture, and then pass the *output* of `.ToString()` on to `FormattableString.Invariant()`.

Reply:
For these two you have to call `ToString()` because they are both `ReadonlySpan<char>`.

For some reason you can't string interpolate these like `$"Foo: {foo}"`

Reply:
With this change, we'd get a CA1305 build error if these were wrong. `ReadonlySpan<char>` doesn't have an overload for `ToString()`:

https://learn.microsoft.com/en-us/dotnet/api/system.readonlyspan-1.tostring?view=net-8.0